### ISSUE TITLE
Gitk doesn`t work

### ISSUE THREAD

**Issue by denysdovhan** [SublimeGit/SublimeGit#178]
When I try to run Gitk from Command Palette, it doesn`t start.
In console I get this:
```
Exception in thread Thread-15:
Traceback (most recent call last):
  File "./threading.py", line 901, in _bootstrap_inner
  File "./threading.py", line 858, in run
  File "SublimeGit/sgit/gitk.py", line 31, in async_inner
TypeError: 'NoneType' object is not callable
```

Can somebody helps me?

**Comment by michaelpedersen** [comment]
Fixed in 1.0.34 which was just released.

**Comment by denysdovhan** [comment]
Thank! That`s work!
